frappe/utils/global_search.py

```diff
@@ -244,6 +244,10 @@ def update_global_search(doc):
 		if doc.get(field.fieldname) and field.fieldtype != "Table":
 			content.append(get_formatted_value(doc.get(field.fieldname), field))
 
+	tag =(frappe.db.get_value(doc.doctype, doc.name, '_user_tags', ignore=1) or '').strip()
+	if tag:
+		content.append("User Tags : " + tag)
```

Comment:
let's not add the label (does not make sense if the base language is not english), just the tag should be fine!

Also can you add a simple test to make sure this works